helios-client/src/test/java/com/spotify/helios/common/JobValidatorTest.java

```diff
@@ -451,4 +452,23 @@ public void testWhitelistedCapabilities_withWhitelist() throws Exception {
     final Set<String> errors2 = validator2.validate(job);
     assertEquals(0, errors2.size());
   }
+
+  @Test
+  public void testImageNamespaceWithHyphens() {
+    final Job job = VALID_JOB.toBuilder()
+        .setImage("b.gcr.io/cloudsql-docker/gce-proxy:1.05 ")
```

Comment:
No, I don't think so, but that was an accident https://github.com/docker/distribution/blob/master/reference/regexp.go#L36